tinygrad/runtime/support/elf.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import annotations
-from typing import Tuple, List, Any
+from typing import Tuple, Dict, List, Any
 from dataclasses import dataclass
 import tinygrad.runtime.autogen.libc as libc
 
 @dataclass(frozen=True)
 class ElfSection: name:str; header:libc.Elf64_Shdr; content:bytes # noqa: E702
 
-def elf_loader(blob:bytes, force_section_align:int=1) -> Tuple[memoryview, List[ElfSection], Any]:
+def elf_loader(blob:bytes, force_section_align:int=1, reserve:int=0) -> Tuple[memoryview, List[ElfSection], Any, Dict[str, int]]:
```

Comment:
this is to reserve space for jump to the entry point instruction

Reply:
This should not be a feature of an ELF loader